src/style.css

```diff
@@ -1 +1,95 @@
 /* add styles here */
+:root {
+  --border-sol-col: 1px solid #e1e7ed;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.form, .form__search {
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.form__search {
+  height: 100%;
+  position: relative;
+}
+
+.img__search{
+  position: absolute;
+  background-image: url("./images/Search.svg");
+}
+
+.form {
+  margin-top: 20px;
+  height: 70px;
+  font-size: 16px;
+  border-radius: 4px;
+  border: var(--border-sol-col);
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61,78,97,.1);
+}
+
+.search__logo--big {
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+}
+
+.logo {
+  position: absolute;
+}
+
+.search__input {
+  width: 100%;
+  font-family: Avenir,sans-serif;
+  padding-left: 62px;
+  border-radius: 3px;
+  border: none;
+  height: 100%;
+  font-size: 16px;
+  font-weight: 400;
+  color: #3d4e61;
+}
+
+.search__input:focus {
+  width: 100%;
+  outline: none;
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+}
```

Comment:
In the **focus** you need to specify only the elements which would change.

Reply:
I don`t quit get what I have to do in this case. Could you explain?